code/game/gamemodes/clock_cult/clock_items.dm

```diff
@@ -540,21 +551,65 @@
 		return 1
 
 
+/obj/item/weapon/ratvars_light //Stuns and blinds the target when struck.
+	name = "Ratvar's light"
+	desc = "A dazzling ball of concentrated light. When clenched in your fist, it's completely invisible."
+	icon = 'icons/effects/clockwork_effects.dmi'
+	icon_state = "ratvars_light"
+	item_state = null
+	w_class = 5
+	flags = NODROP
+	force = 5 //Also serves as a weak melee weapon!
+	damtype = BURN
+	hitsound = 'sound/weapons/sear.ogg'
+	attack_verb = list("scorched", "seared", "burnt")
+
+/obj/item/weapon/ratvars_light/examine(mob/user)
+	..()
+	user << "<span class='brass'>Attack a human to stun and blind them.</span>"
+
+/obj/item/weapon/ratvars_light/afterattack(atom/target, mob/living/user, flag, params)
+	if(!is_servant_of_ratvar(user))
+		user << "<span class='heavy_brass'>\"[text2ratvar("You seem keen on using that light.")]\"</span>"
+		target = user
+	else
+		if(!ishuman(target))
+			return
+	var/mob/living/carbon/human/victim = target
+	user.visible_message("<span class='warning'>[user]'s hand bursts with blinding light!</span>", "<span class='danger'>You release the power of [src]!</span>")
+	victim.Stun(10)
+	victim.Weaken(10)
+	victim.flash_act(1,1)
+	if(!iscultist(victim))
+		victim << "<span class='userdanger'>Blinding light and a soothing voice fill your mind...</span>"
+		victim << "<span class='inathneq'>\"[text2ratvar("Let Engine's light fill your body. Be purged of impure thoughts.")]\"</span>"
+		victim.silent += 5
+		victim.confused += 15
+	else
+		victim << "<span class='userdanger'>Searing light and a malevolent voice fill your mind...</span>"
+		victim << "<span class='inathneq'>\"[text2ratvar("Blood-spilling heretic! I hope this one rips out your black heart!")]\"</span>"
+		victim.silent += 10
+		victim.confused += 30
+		victim.cultslurring += 15
+		victim.adjustFireLoss(15)
+	qdel(src)
+
+
 /obj/item/clothing/head/helmet/clockwork //Clockwork armor: High melee protection but weak to lasers
 	name = "clockwork helmet"
 	desc = "A heavy helmet made of brass."
 	icon = 'icons/obj/clothing/clockwork_garb.dmi'
 	icon_state = "clockwork_helmet"
 	w_class = 3
 	resistance_flags = FIRE_PROOF | ACID_PROOF
-	armor = list(melee = 80, bullet = 50, laser = -15, energy = 0, bomb = 35, bio = 0, rad = 0, fire = 100, acid = 100)
+	armor = list(melee = 50, bullet = 50, laser = -15, energy = 0, bomb = 35, bio = 0, rad = 0, fire = 100, acid = 100)
```

Comment:
I'm not sure why you're nerfing the melee armor of this, given the laser weakness it has...